src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -3,30 +3,62 @@
 #include "App.hpp"
 #include "Hook.hpp"
 #include "Systems/ScriptCompilationSystem.hpp"
+#include <winbase.h>
```

Comment:
What does this header bring? If needed 100%, move it to the `stdafx.hpp`.

Reply:
Sorry, I think that might have been added automatically - I'll see if we can do without it.